code/src/index.css

```diff
@@ -1,13 +1,382 @@
     monospace;
+    color: var(--first);
+}
+
+
+/**********LOADER************/
+
+@keyframes spin {
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.loader-overlay {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: #fff;
+}
+
+.loader-spinner {
+  display: inline-block;
+  width: 100px;
+  border-width: 50px;
+  border-top-style: solid;
+  border-bottom-style: solid;
+  border-top-color: #fff;
+  border-bottom-color: var(--second);
+  border-radius: 50%;
+  animation: spin 3s linear infinite;
+}
+
+
+/**********CARDS LAYOUT************/
+
+.thoughts-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 350px;
+  margin: 0 auto;
+}
+
+form {
+  border: 2px solid var(--first);
+  width: 90%;
+  height: fit-content;
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+  margin-bottom: 1rem;
+  padding: 1rem;
+  box-shadow: 6px 6px var(--first);
+  background-color: #fff;
+}
+
+textarea {
+  width: 90%;
+  padding: 10px;
+  font-family: 'Space Mono', monospace;
+  border: 1px solid var(--first);
+}
+
+textarea:hover,
+textarea:focus {
+  border: 1px solid var(--second);
+  outline: none;
+}
+
+.thought-container {
+  width: 90%;
+  border: 2px solid var(--first);
+  height: 140px;
+  margin: 1rem 0;
+  padding: 1rem;
+  box-shadow: 6px 6px var(--first);
+  overflow-wrap: break-word;
+  font-family: 'Space Mono', monospace;
+  background-color: #fff;
+}
+
+.message {
+  height: 60px;
+  padding: 1rem 0;
+  overflow: scroll;
+}
+
+.like-and-date-wrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.like {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.send-heart {
+  margin: 8px;
+  font-size: 1rem;
+}
+
+.heart {
+  font-size: 1.4rem;
+  position: absolute;
+  right: 10px;
+  top: 9px;
+}
+
+.date {
+  display: flex;
+  align-items: center;
+}
+
+
+/**********FOOTER************/
+
+footer {
+  margin-top: auto;
+  text-align: center;
+  padding: 1rem;
+}
+
+
+/**********TEXT AND TITLES************/
+
+.form-title {
+  font-size: 1.1rem;
+  font-weight: normal;
+  margin-bottom: 1rem;
+  font-family: 'Space Mono', monospace;
+}
+
+.footer-text {
+  font-family: 'Space Mono', monospace;
+  font-size: 0.8rem;
+  color: var(--first);
+  margin: 0;
+  padding: 0;
+}
+
+
+/**********BUTTONS************/
+
+.button-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  align-items: flex-end;
+  margin-top: 15px;
+  font-family: 'Space Mono', monospace;
+}
+
+.submit-button {
+  border-radius: 20px;
+  border: none;
+  background-color: var(--third);
+  width: fit-content;
+  height: 40px;
+  padding: 3px;
+  box-shadow: 0px 10px 10px -7px #d87093;
+  font-size: 0.8rem;
+  font-family: 'Space Mono', monospace;
+}
```

Comment:
I can see that you added the font-family in a few places in your code, didn't it work to have it only in the "code" that you have in the beginning?